src/frontend/contexts/ProjectContext.tsx

```diff
@@ -2,11 +2,12 @@ import * as React from 'react';
 import {type MapeoClientApi} from '@mapeo/ipc';
 import {useApi} from './ApiContext';
 import {usePersistedProjectId} from '../hooks/persistedState/usePersistedProjectId';
+import {Loading} from '../sharedComponents/Loading';
 
 type MapeoProject = Awaited<ReturnType<MapeoClientApi['getProject']>>;
 
 type ActiveProjectContextType = {
-  project: MapeoProject | undefined;
+  project: MapeoProject;
```

Comment:
I made it so that the context returns a loading state when undefined, and only renders its children when the mapeo project has been initialized. this allows us to use `useSuspenseQuery` which doesn't have an `enabled` property